ui/components/ui/editable-label/editable-label.js

```diff
@@ -71,6 +71,7 @@ export default class EditableLabel extends Component {
         <ButtonIcon
           iconName={IconName.Check}
           onClick={() => this.handleSubmit(isValidAccountName)}
+          data-testid="save-account-label-input"
```

Comment:
is it correct to name this id as `save-account-label-input` since it is identifying the button itself and not an input field?

Reply:
Fair point, thank you.  Will rename this more accurately in the future.